### PR TITLE
Porting SublimeLinter-eslint stdin logic

### PR DESCRIPTION
**Pull request by doctyper**
Porting SublimeLinter-eslint stdin logic to SublimeLinter-contrib-eslint_d. Closes #5. Depends on mantoni/eslint_d.js#15

### CODE REVIEW

**Comment by roadhump** [comment]
Wow, great.
You should wait until eslint_d is updated with `stdin` support and then change `version` attribute of plugin to the latest version of eslint_d.

**Comment by doctyper** [comment]
Will do.

**Comment by doctyper** [comment]
@roadhump :+1:

**Comment by doctyper** [comment]
@roadhump Let me know if there's anything else you need.

File: README.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ Before using this plugin, you must ensure that `eslint_d` is installed on your s
 
 1. Install `eslint_d` by typing the following in a terminal:
    ```
-   npm install -g eslint_d
+   npm install -g eslint_d@>=2.3.1
```
Comment (roadhump) [inline README.md]: I don't think it should be version here and below, it will be too confusing for users, let npm just to install latest version.

**Pull request merged**